app/viewmodels/IndexPageHelper.scala

```diff
@@ -210,4 +212,16 @@ class IndexPageHelper @Inject()(p: views.html.components.p,
       Some(messages(msgKey))
     } else None
   }
+
+  private def returnEstimatedVATMessageIfMoreThanZero(vatCharges: BigDecimal, isEstimatedVAT: Boolean, msgKeyToApply: String)(implicit messages: Messages): Option[String]={
```

Comment:
I'd be careful on the naming here - `vatCharges` should be `vatInterest`:
```suggestion
  private def returnEstimatedVATMessageIfMoreThanZero(vatInterest: BigDecimal, isEstimatedVAT: Boolean, msgKeyToApply: String)(implicit messages: Messages): Option[String]={
```